en/docs/publish/api-microgateway/quick-start-guide.md

```diff
@@ -0,0 +1,96 @@
+    ```
+
+    The project is now initialized. You should notice a directory with name "petstore" being created in the location 
+    where you executed the command. 
+
+2.  Now lets start the microgateway on docker by executing the docker compose script inside the `MG_HOME`. 
+    Navigate to `MG_HOME` and execute the following command.
+
+    ``` bash
+    docker-compose up -d
+    ```
+    
+    Once containers are up and running, we can monitor the status of the containers using the following command
+    
+    ``` bash
+    docker ps | grep mg-
+    ```
+
+3.  Now let's deploy our first API to Microgateway using the project created in the step 3. 
+    Navigate to the location where the petstore project was initialized.
+    Execute the following command to deploy the API in the microgateway.
+       
+    ``` bash
+    apictl mg deploy --host https://localhost:9843 --file petstore  -u admin -p admin -k
+    ```
+    
+    !!! note  
+        The user credentials can be configured in the configurations of the `MG_HOME` distribution. 
+        `admin:admin` is the default accepted credentials by the microgateway adapter.
+        Go to `MG_HOME/resources/conf/config.toml` and modify as below.
+        
+        ``` toml
+        [server]
+        username = "admin"
+        password = "admin"
+        ``` 
+        
+### Step 3 - Invoke the API
+
+1.  The next step would be to invoke the API using a REST tool. Since APIs on the Microgateway are by default secured. 
+    We need a valid token in order to invoke the API. 
+    Use the following sample token accepted by the microgateway to access the API. 
+    Lets set the token to command line as a variable.
+    
+    ``` bash
+    TOKEN=eyJ4NXQiOiJNell4TW1Ga09HWXdNV0kwWldObU5EY3hOR1l3WW1NNFpUQTNNV0kyTkRBelpHUXpOR00wWkdSbE5qSmtPREZrWkRSaU9URmtNV0ZoTXpVMlpHVmxOZyIsImtpZCI6Ik16WXhNbUZrT0dZd01XSTBaV05tTkRjeE5HWXdZbU00WlRBM01XSTJOREF6WkdRek5HTTBaR1JsTmpKa09ERmtaRFJpT1RGa01XRmhNelUyWkdWbE5nX1JTMjU2IiwiYWxnIjoiUlMyNTYifQ==.eyJhdWQiOiJBT2syNFF6WndRXzYyb2QyNDdXQnVtd0VFZndhIiwic3ViIjoiYWRtaW5AY2FyYm9uLnN1cGVyIiwibmJmIjoxNTk2MDA5NTU2LCJhenAiOiJBT2syNFF6WndRXzYyb2QyNDdXQnVtd0VFZndhIiwic2NvcGUiOiJhbV9hcHBsaWNhdGlvbl9zY29wZSBkZWZhdWx0IiwiaXNzIjoiaHR0cHM6Ly9sb2NhbGhvc3Q6OTQ0My9vYXV0aDIvdG9rZW4iLCJrZXl0eXBlIjoiUFJPRFVDVElPTiIsImV4cCI6MTYyNzU0NTU1NiwiaWF0IjoxNTk2MDA5NTU2LCJqdGkiOiIyN2ZkMWY4Ny01ZTI1LTQ1NjktYTJkYi04MDA3MTFlZTJjZWMifQ==.otDREOsUUmXuSbIVII7FR59HAWqtXh6WWCSX6NDylVIFfED3GbLkopo6rwCh2EX6yiP-vGTqX8sB9Zfn784cIfD3jz2hCZqOqNzSUrzamZrWui4hlYC6qt4YviMbR9LNtxxu7uQD7QMbpZQiJ5owslaASWQvFTJgBmss5t7cnurrfkatj5AkzVdKOTGxcZZPX8WrV_Mo2-rLbYMslgb2jCptgvi29VMPo9GlAFecoMsSwywL8sMyf7AJ3y4XW5Uzq7vDGxojDam7jI5W8uLVVolZPDstqqZYzxpPJ2hBFC_OZgWG3LqhUgsYNReDKKeWUIEieK7QPgjetOZ5Geb1mA==
+    ``` 
+
+2.  We can now invoke the API running on the microgateway using cURL as below.
+
+    ``` bash
+    curl -X GET "https://localhost:9095/v2/pet/1" -H "accept: application/json" -H "Authorization:Bearer $TOKEN" -k
```

Comment:
shall we change this to findby status resource if possible, because sometime petId:1 returns 404

Reply:
Fixed